airflow_prometheus_exporter/xcom_config.py

```diff
@@ -1,13 +1,10 @@
 import yaml
-from pathlib import Path
 
-CONFIG_FILE = Path.cwd() / "config.yaml"
 
-
-def load_xcom_config():
+def load_xcom_config(path):
     """Loads the XCom config if present."""
     try:
-        with open(CONFIG_FILE) as file:
+        with open(path) as file:
```

Comment:
The path is the path to the config file or just the folder? If it is the config file, I would rename the varible to `config_path`